web/packages/teleport/src/Navigation/SideNavigation/Navigation.tsx

```diff
@@ -75,10 +76,14 @@ const PanelBackground = styled.div`
 export type NavigationSection = {
   category?: SidenavCategory;
   subsections?: NavigationSubsection[];
+  route?: string;
   /* standalone is whether this is a clickable nav section with no subsections/drawer. */
   standalone?: boolean;
+  /* Icon is the custom icon to display for a standalone section. This should only for standalone sections, as icons for categories are derived automatically by CategoryIcon */
+  Icon?: (props) => ReactNode;
+  /* title is the custom title of a standalone section */
+  title?: string;
```

Comment:
maybe we should change standalone to instead be an optional object that requires two properties, Icon and title. 

and then something is "standalone" is that object exists. This will keep us more type safe (not allowing a standalone to exist without icons or titles) and be a bit more readable (if i saw Icon, i wouldnt know it was for standalone only if i didnt read the docs/comments)

Reply:
great idea